Decred Wallet/es.lproj/Localizable.strings

```diff
@@ -395,14 +395,17 @@
 "forMoreInformationPleaseVisit" = "Para más información por favor visita";
 
 /* Settings */
-"spendingPinPassInfo" = "Requerido para enviar fondos.";
-"startupPinPass" = "PIN/Contraseña de inicio";
-"changeStatupPinPass" = "Cambiar PIN/Contraseña de inicio";
-"startupPinPassInfo" = "Requerido para ingresar a la app.";
-"spendUnconfirmedFund" = "Spend Unconfirmed Funds";
+"spendingPinPassInfo" = "Required to send funds.";
+"startupPinPass" = "Startup PIN/Password";
+"changeStatupPinPass" = "Change startup PIN/Password";
+"startupPinPassInfo" = "Required to enter app.";
+"spendUnconfirmedFund" = "Spend unconfirmed funds";
```

Comment:
Why are we changing these to English? The English version only changes capitalization, actual text remains valid.

Reply:
capitalization matters for display in the app.